app/templates/components/application-footer.hbs

```diff
@@ -13,6 +13,8 @@
       <ListLinks @links={{data.resources_links}} />
       <h4>Community</h4>
       <ListLinks @links={{data.community_links}} />
+      <h4>AVAILABLE PREFIXES</h4>
```

Comment:
I am not sure if this is part of the Specification?

Reply:
It is not.  This was for my debugging.  Deleting this.